runtime/oti/vmargs_api.h

```diff
@@ -217,6 +217,17 @@ addJavaPropertiesOptions(J9PortLibrary * portLib, J9JavaVMArgInfoList *vmArgumen
 IDATA
 addJarArguments(J9PortLibrary * portLib, J9JavaVMArgInfoList *vmArgumentsList, const char *jarPath, J9ZipFunctionTable *zipFuncs, UDATA verboseFlags);
 
+/**
+ * Add the arguments coming from a specific environment variable
+ * @param portLib port library
+ * @param envVarName environment variable name
+ * @param vmArgumentsList current list of arguments
+ * @param verboseFlags set to VERBOSE_INIT for verbosity
+ * @return 0 on success, negative value on failure
+ */
+IDATA
+addEnvironmentVariableArguments(J9PortLibrary* portLib, const char* envVarName, J9JavaVMArgInfoList *vmArgumentsList, UDATA verboseFlags);
```

Comment:
Please format consistently (`*` binds more tightly to parameter name than the type, like `vmArgumentsList`); use same format in `vmargs.c`.

Reply:
Updated.